pkgs/development/libraries/arrayfire/default.nix

```diff
@@ -245,5 +245,6 @@ stdenv.mkDerivation rec {
       chessai
       twesterhout
     ];
+    broken = true;
```

Comment:
That's a regression from master. We should at least add a comment with a short reason.

Reply:
It's not? It doesn't build on master either.

Reply:
(and hasn't for ages)